src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConversionExpression.cs

```diff
@@ -2816,8 +2816,8 @@ class C2
     Variables: Local_1: C2 c2
     Initializer: IConversionExpression (Implicit, TryCast: False, Unchecked) (OperatorMethod: C2 C2.op_Implicit(System.Int64 c1)) (OperationKind.ConversionExpression, Type: C2) (Syntax: 'i1')
         Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: True) (MethodSymbol: C2 C2.op_Implicit(System.Int64 c1))
-        Operand: IConversionExpression (Implicit, TryCast: False, Unchecked) (OperatorMethod: C2 C2.op_Implicit(System.Int64 c1)) (OperationKind.ConversionExpression, Type: System.Int64) (Syntax: 'i1')
```

Comment:
>OperatorMethod: C2 C2.op_Implicit(System.Int64 c1) [](start = 78, length = 50)

This shows why it is wrong to request conversion information from semantic model based on syntax, several nodes can be associated with  the same syntax and represent distinct conversions. The question is why this wasn't caught earlier? We have tests, but for some reason they do not help.